ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -314,7 +314,9 @@ function AuthProviderForm({
                                             isDisabled={action === 'edit'}
                                             isSmall
                                         >
-                                            {selectedAuthProvider.active
+                                            {selectedAuthProvider.active ||
+                                            selectedAuthProvider.traits?.mutabilityMode ===
+                                                'ALLOW_FORCED'
```

Comment:
```suggestion
                                            selectedAuthProvider.traits?.mutabilityMode !==
                                                'ALLOW'
```
Please change all the conditions accordingly, it is very to confusing to see the condition in the original form (who says the other option isn't `DONT_ALLOW`?)